core/assets/bundles/bundle_id_ID.properties

```diff
@@ -1981,35 +1996,35 @@ hint.waveFire = [accent]Wave[] yang terisi dengan air akan memadamkan air dalam
+gz.research = Buka \ue875 pohon teknologi.\nRiset \uf870 [accent]Bor Mekanis[], lalu pilih dari menu di kanan bawah.\nKlik pada tambalan tembaga untuk menempatkannya.
+gz.research.mobile = Buka \ue875 pohon teknologi.\nRiset \uf870 [accent]Bor Mekanis[], lalu pilih dari menu di kanan bawah.\nKetuk pada tambalan tembaga untuk menempatkannya.\n\nKetuk \ue800 [accent]tanda centang[] di kanan bawah untuk mengonfirmasi.
 gz.conveyors = Riset dan tempatkan \uf896 [accent]konveyor[] untuk memindahkan sumber daya yang ditambang\ndari bor ke inti.\n\nKlik dan seret untuk menempatkan beberapa konveyor.\n[accent]Gulir[] untuk memutar arah konveyor
 gz.conveyors.mobile = Riset dan tempatkan \uf896 [accent]konveyor[] untuk memindahkan sumber daya yang ditambang\ndari bor ke inti.\n\nTahan jari Anda sebentar dan seret untuk menempatkan beberapa konveyor.
-gz.drills = Perluas operasi penambangan.\ntempatkan lebih banyak Bor Mekanik.\nTambang 100 tembaga.
+gz.drills = Perluas operasi penambangan.\ntempatkan lebih banyak Bor Mekanis.\nTambang 100 tembaga.
 gz.lead = \uf837 [accent]Timah[] adalah sumber daya lain yang umum digunakan.\nSiapkan bor untuk menambang timah.
 gz.moveup = \ue804 Bergerak ke atas untuk objektif lebih lanjut.
-gz.turrets = Riset dan tempatkan 2 menara \uf861 [accent]Duo[] untuk mempertahankan inti.\nMenara Duo membutuhkan \uf838 [accent]amunisi[] dari konveyor.
+gz.turrets = Riset dan tempatkan 2 menara \uf861 [accent]Duo[] untuk mempertahankan inti.\nMenara Duo membutuhkan amunisi \uf838 [accent]tembaga[] dari konveyor.
 gz.duoammo = Suplai menara Duo dengan [accent]tembaga[], menggunakan konveyor.
 gz.walls = [accent]Dinding[] dapat menahan kerusakan yang mencapai bangunan.\nTempatkan \uf8ae [accent]dinding tembaga[] di sekitar menara.
 gz.defend = Musuh datang, bersiaplah untuk bertahan.
-gz.aa = Unit terbang tidak dapat dengan mudah dibunuh dengan menara standar.Menara\n\uf860 [accent]Scatter[] memberikan anti-udara yang sangat baik, tetapi membutuhkan \uf837 [accent]timah[] sebagai amunisi.
+gz.aa = Unit terbang tidak dapat dengan mudah dibunuh dengan menara standar.\nMenara \uf860 [accent]Scatter[] memberikan anti-udara yang sangat baik, tetapi membutuhkan \uf837 [accent]timah[] sebagai amunisi.
 gz.scatterammo = Suplai Menara Scatter dengan [accent]timah[], menggunakan konveyor.
 gz.supplyturret = [accent]Suplai Menara
 gz.zone1 = Ini adalah zona pendaratan musuh.
-gz.zone2 = Apa pun yang dibangun dalam radius tersebut akan hancur ketika gelombang mulai.
-gz.zone3 = Gelombang akan dimulai sekarang.Bersiap.
+gz.zone2 = Apa pun yang dibangun dalam radius tersebut \nakan hancur ketika gelombang mulai.
+gz.zone3 = Gelombang akan dimulai sekarang. Bersiap.
 gz.finish = Bangun lebih banyak menara, tambang lebih banyak sumber daya,\ndan bertahan melawan semua gelombang [accent]untuk menaklukkan sektor[].
```

Comment:
*melawan?*
I prefer *melewati* if it contexted to defending through all waves.
and also *menghadapi (terhadap)* is usable context from original string/entry